main/src/modules/reactiveLayer/homeostasis/include/homeostasisManagerIcub.h

```diff
@@ -21,6 +21,10 @@ class HomeostaticModule: public RFModule
     std::string moduleName;
     double period;
 
+    double stress;
```

Comment:
Because you want to Keep It Simple and Stupid :). Global variables should only ever be used if they are used across different methods, and should be avoided by all other means. If you can do things locally, do them locally. This way, it's much easier to debug code; initialization etc. is done at the place where the variable is being used, so the code is easier to understand. So many advantages :+1: